dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,106 @@
+class SentryStackFrame {
+  static final SentryStackFrame asynchronousGapFrameJson = SentryStackFrame()
+    ..absPath = '<asynchronous suspension>';
+
+  List<String> preContext;
+  List<String> postContext;
+  Map<String, String> vars;
+  List<int> framesOmitted;
+  String filename;
+  String function;
+  String module;
+  String lineno;
+  String colno;
+  String absPath;
+  String contextLine;
+  bool inApp;
+  String package;
+  bool native;
+  String platform;
+  String imageAddr;
+  String symbolAddr;
+  String instructionAddr;
+  String rawFunction;
+
+  Map<String, dynamic> toJson([String origin = '']) {
+    final json = <String, dynamic>{};
+
+    if (preContext?.isNotEmpty ?? false) {
```

Comment:
most of our NotEmpty checks use:

`field != null && field.isNotEmpty`

guess `field?.isNotEmpty ?? false` is more elegant, should we change in the other places too, I dont like the idea of mixing things up, we could decide the best approach and go for it

Reply:
✅  `field != null && field.isNotEmpty` is probably clearer